### DISCUSSION TITLE
SMTP configuration

### DISCUSSION THREAD

**Discussion by krseguros** [YetiForceCompany/YetiForceCRM#17246] [How to]
Hello,

Please help with the SMTP configuration, I have tried in several ways and I have not been able to configure it, what is the appropriate configuration? I manage emails through office365
![image](https://github.com/YetiForceCompany/YetiForceCRM/assets/96985476/7c51c487-559b-4a37-b77e-32692becde25)
![image](https://github.com/YetiForceCompany/YetiForceCRM/assets/96985476/3a349110-3b0d-4533-badc-3e63bcc47e84)
![image](https://github.com/YetiForceCompany/YetiForceCRM/assets/96985476/2129aff8-318d-4c7b-97d2-a5ddcf663064)
¡¡¡Error!!!
SMTP ERROR: Password command failed: 535 5.7.139 Authentication unsuccessful, the request did not meet the criteria to be authenticated successfully. Contact your administrator. [MN2PR07CA0028.namprd07.prod.outlook.com 2023-08-13T23:04:58.529Z 08DB9B1CD2F483ED] SMTP Error: Could not authenticate. SMTP connect() failed. https://github.com/PHPMailer/PHPMailer/wiki/Troubleshooting ######################################## SMTP connect() failed. https://github.com/PHPMailer/PHPMailer/wiki/Troubleshooting
![image](https://github.com/YetiForceCompany/YetiForceCRM/assets/96985476/91e88cf0-ddb2-4fd8-a501-5605d66623ea)

**Comment by rskrzypczak** [comment]
Remove the imap configuration.
An imap connection requires oauth2, and yetiforce does not support it in the current version.

**Reply by krseguros** [reply]
It is not clear to me, how do I do this?

**Comment by rskrzypczak** [comment]
You can't use imap: 
![image](https://github.com/YetiForceCompany/YetiForceCRM/assets/10221999/5f624519-011b-4b67-9da4-865096be6dc4)
so uncheck it.

**Reply by krseguros** [reply]
Even after unchecking this option, it does not accept the configuration. What can be wrong here?
![image](https://github.com/YetiForceCompany/YetiForceCRM/assets/96985476/14784e13-383f-4785-860e-a4509333e09f)

![image](https://github.com/YetiForceCompany/YetiForceCRM/assets/96985476/f7a2112c-8541-46d5-a871-3b82d331a256)

**Reply by Giorgos-Hellene** [reply]
Try with ssl Safety protocol* instead of tls

![εικόνα](https://github.com/YetiForceCompany/YetiForceCRM/assets/50912403/9527f13c-de69-4830-b834-7f20c3884002)

**Comment by ghost** [comment]
You may need to enable smtp in the office365 admin panel, it was off by default for me.

[https://www.oryon.net/knowledge-base/article/how-to-enable-smtp-for-an-user-in-office-365/](https://www.oryon.net/knowledge-base/article/how-to-enable-smtp-for-an-user-in-office-365/)
